### PR TITLE
feat: wrap apps with theme, include extensions and add docs for the hook

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,3 +123,11 @@ During dev, we do not get any inline styles from Gatsby – they are handled by
 - During runtime, we need to ensure that our link with the id `eufemia-style-theme` is placed after `commons.css`. We do that with `headElement.appendChild(styleElement)`
 - Use `uniqueId` to reload css files as there is not unique build hash, unlike we get during production
 - Use `MutationObserver` to reload the current theme file, because Webpack uses hot module replacement, so we need to reload as well
+
+## Releases
+
+Releases are med with [semantic-release](https://github.com/semantic-release/semantic-release).
```
Comment (langz) [inline README.md]:
```suggestion
Releases are made with [semantic-release](https://github.com/semantic-release/semantic-release).
```

File: README.md [hunk 2]
```diff
@@ -96,3 +123,11 @@ During dev, we do not get any inline styles from Gatsby – they are handled by
...
+
+Decorate your pull requests with either `fix: your release message` or `feat: your release message`.
+
+When a pull request is merged into the `main` branch, a new release will be published.
```
Comment (langz) [inline README.md:141]: 🚀

**Review by langz — APPROVED** [review]

**Comment by tujoworker** [comment]
It took some time to get the CI up and running again. Not sure why it did not work anymore. But it works now that we call the playwright install within the workspace, where we run the test:

`yarn workspace e2e playwright install`

File: packages/gatsby-plugin-eufemia-theme-handler/themeHandler.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+import { Theme } from '@dnb/eufemia/shared'
```
Comment (langz) [inline packages/gatsby-plugin-eufemia-theme-handler/themeHandler.js:2]: Seem like this import causes the following issue:
```
 ERROR #98124  WEBPACK.DEVELOP-HTML

Generating development SSR bundle failed

Can't resolve '@dnb/eufemia/shared' in '/Users/anderslangseth/dev/eufemia/node_modules/gatsby-plugin-eufemia-theme-handler'

If you're trying to use a package make sure that '@dnb/eufemia/shared' is installed. If you're trying to use a local file make sure that the path is correct.

File: ../../node_modules/gatsby-plugin-eufemia-theme-handler/themeHandler.js:2:0
```

Also reported here: https://github.com/dnbexperience/eufemia/pull/2671/files#r1339666868

**Pull request merged**